presto-hive/src/main/java/com/facebook/presto/hive/HiveStagingFileCommitter.java

```diff
@@ -49,7 +49,12 @@ public HiveStagingFileCommitter(
     }
 
     @Override
-    public ListenableFuture<Void> commitFiles(ConnectorSession session, String schemaName, String tableName, List<PartitionUpdate> partitionUpdates)
+    public ListenableFuture<Void> commitFiles(
+            ConnectorSession session,
+            String schemaName,
+            String tableName,
```

Comment:
Maybe change the order of arguments to group table related arguments together, e.g.:

```
String schemaName,
String tableName,
Path tablePath,
```